erpnext_com/www/docs/user/manual/en/loan-management/loan-security-price.md

```diff
@@ -0,0 +1,32 @@
+<!-- add-breadcrumbs -->
+# Loan Security Price
+> Introduced in Version 13
+
+**Loan Security Price is the record in which you can log the loan security price and its validity.**
+
+During a loan security price update process, Loan Security Price is used to fetch the valid loan security
+price for updation
+
+To access the Loan Security Price list, go to:
+> Home > Loan Management > Loan Security > Loan Security Price
+
+
+<img class="screenshot" alt="Loan Security Price" src="{{docs_base_url}}/assets/img/loan-management/loan-security-price.png">
```

Comment:
what is the America/New_York label?

Reply:
It is the timestamp field label added by default based on the region. My test site has USA as country

Reply:
Why does New_York have an underscore?